src/reconScreens/ReconOnboarding/ReconOnbdLanding.res

```diff
@@ -1,4 +1,27 @@
 @react.component
 let make = () => {
-  <div> {"OnBoarding"->React.string} </div>
+  open PageUtils
+
+  let handleGetStarted = (): unit => {
+    Js.log("Get Started")
+  }
+
+  <div className="flex flex-col w-full gap-8 items-center justify-center">
+    <PageHeading
+      customHeadingStyle="gap-3 max-w-[860px] flex flex-col items-center"
```

Comment:
use predefined class names or define this max-w-[860px] value on taiwind cnfig file and use it